README.md

```diff
@@ -6,7 +6,7 @@
 
 <div align="center">
   <a href="https://github.com/babel/babel">
-    <img src="https://rawgit.com/babel/logo/master/babel.svg" alt="Babel logo" width="200" height="200">
+    <img src="https://raw.githubusercontent.com/babel/logo/master/babel.svg" alt="Babel logo" width="200" height="200">
```

Comment:
rawgit has reached eol in 2018. I am surprised that it still works in 2024. Anyway we can just serve the logo from the GitHub CDN.